src/tests/functional/shared_test_classes/src/single_op/gather.cpp

```diff
@@ -316,5 +316,54 @@ void GatherStringWithIndicesDataLayerTest::generate_inputs(const std::vector<ov:
     inputs.insert({data_input.get_node_shared_ptr(), ov::Tensor(element::string, data_input.get_shape(), string_data.data())});
 }
 
+std::string GatherMixedPrecLayerTest::getTestCaseName(const testing::TestParamInfo<gatherMixPrecParamsTuple>& obj) {
+    auto [ shapes, indices_shape, axis_batch_idx, model_type, out_type, device_name ] = obj.param;
+    std::ostringstream result;
+    result << "IS=(";
+    for (size_t i = 0lu; i < shapes.size(); i++) {
+        result << ov::test::utils::partialShape2str({shapes[i].first}) << (i < shapes.size() - 1lu ? "_" : "");
+    }
+    result << ")_TS=";
+    for (size_t i = 0lu; i < shapes.front().second.size(); i++) {
+        result << "{";
+        for (size_t j = 0lu; j < shapes.size(); j++) {
+            result << ov::test::utils::vec2str(shapes[j].second[i]) << (j < shapes.size() - 1lu ? "_" : "");
+        }
+        result << "}_";
+    }
```

Comment:
`ov::test::InputShape` has `operator<<`, please use it

Reply:
that's great, done, thanks Vladislav!